rasa_nlu/data_router.py

```diff
@@ -4,25 +4,44 @@
 from __future__ import unicode_literals
 
 import datetime
+
 import glob
 import json
 import logging
-import multiprocessing
 import os
 import tempfile
+import io
 
 from builtins import object
 from typing import Text
 
+from concurrent.futures import ProcessPoolExecutor as ProcessPool
+from twisted.internet import defer
+from twisted.logger import jsonFileLogObserver, Logger
+
 from rasa_nlu import utils
 from rasa_nlu.components import ComponentBuilder
 from rasa_nlu.config import RasaNLUConfig
 from rasa_nlu.model import Metadata, InvalidModelError, Interpreter
-from rasa_nlu.train import do_train
+from rasa_nlu.train import do_train_in_worker
 
 logger = logging.getLogger(__name__)
 
 
+def deferred_from_future(future):
```

Comment:
Right, as it is an internal function and the name sounded explicit for me I was hoping for you not to no notice. But I'll fix that 😄